tests/README.md

```diff
@@ -0,0 +1,81 @@
+## Backend development
```

Comment:
This will file as-is I don't think it will be included as part of the [documentation](https://treeherder.readthedocs.io/).